TombEngine/Shaders/DX11_Sprites.fx

```diff
@@ -15,7 +15,7 @@ struct PixelShaderInput
 	float3 Normal: NORMAL;
 	float2 UV: TEXCOORD1;
 	float4 Color: COLOR;
-	float Fog : FOG;
+	float  Fog : FOG;
```

Comment:
One white space too much

Reply:
It is intentional to beautify struct alignment :)